test/domain/todo.ts

```diff
@@ -1,28 +1,30 @@
-import { AggregateRoot, DomainHelper, DomainContext, DomainEvent, DomainEventData} from "../../src";
-import { TodoState, TodoStateFactory } from "./todo-state";
-import { TodoCreated } from "./events/todo-created";
-import { TodoTitleUpdated } from "./events/todo-title-updated";
-import { TodoDescriptionUpdated } from "./events/todo-description-updated";
-import { TodoMarkedAsCompleted } from "./events/todo-marked-as-completed";
 import { given } from "@nivinjoseph/n-defensive";
-import { TodoDescription } from "./value-objects/todo-description";
-import { TodoDomainEvent } from "./events/todo-domain-event";
-import { TodoRebased } from "./events/todo-rebased";
+import { AggregateRoot, DomainContext, DomainEvent, DomainEventData, DomainHelper } from "../../src/index.js";
+import { TodoCreated } from "./events/todo-created.js";
+import { TodoDescriptionUpdated } from "./events/todo-description-updated.js";
+import { TodoDomainEvent } from "./events/todo-domain-event.js";
```

Comment:
Cant we just import the type here?

Reply:
You have seen it blow up when set up like that?

Reply:
Yes